.gitignore

```diff
@@ -33,3 +33,4 @@ GeneratedPluginRegistrant.java
 build/
 .flutter-plugins
 
+UserAgent.h
```

Comment:
Can remove this change once switching to the NSBundle based approach (see comments below)

Reply:
I think I prefer to put this in a plugin-specific .gitignore rather than the root of the first-party plugins repo since it feels sort of like an implementation detail of a subset of the plugins.

Reply:
Used plugin specific .gitignore